odc-product-delete/README.md

```diff
@@ -0,0 +1,60 @@
+# Delete ODC Product (In beta)
+
```

Comment:
Please add a big, obvious "Warning" section at the top.

It's not dangerous because it's in beta, it's dangerous because it deletes data and it should be used very carefully!

Reply:
Added Warning text